### ISSUE TITLE
i18n.dart constantly regenerated

### ISSUE THREAD

**Issue by rostopira** [long1eu/flutter_i18n#112]
I think this started happening when I created another flutter project inside current project directory

P.S.: GIF is looped, but it actually happens every second

![scr](https://user-images.githubusercontent.com/16724952/66814549-710a7680-ef3f-11e9-83ef-222018c3507f.gif)

**Comment by noordawod** [comment]
The plugin doesn't support multi module projects, right @long1eu?

**Comment by long1eu** [comment]
True. Only the cli generator knows about multiple projects.

**Comment by rostopira** [comment]
I'm not asking to support multi-module, I ask only for this bug :D
I would actually prefer manual mode, like press button to regenerate, cause I don't modify my arb files every minute, and need it actually in single project out of tens

**Comment by long1eu** [comment]
@rostopira makes sense. Can you try to use the generator?https://github.com/long1eu/flutter_i18n/tree/master/flutter_l10n

**Comment by liugangnhm** [comment]
same issue

**Comment by liugangnhm** [comment]
> same issue

I disabled the `Save Actions` plugin, the issue was gone. 💯

**Comment by rookie-12** [comment]
where is `Save Actions` plugin option?

**Comment by liugangnhm** [comment]
![image](https://user-images.githubusercontent.com/743845/69592305-825f9c00-1030-11ea-9691-b429dca531ff.png)


> where is `Save Actions` plugin option?

@rookie-12

**Comment by Yarikk26** [comment]
Hi there. Same on AndroidStudio 3.5.3. Marked generated file as read-only. Maybe  
[stacktrace](https://github.com/long1eu/flutter_i18n/files/3957742/stacktrace.txt) will be helpful.
